translations/he/LC_MESSAGES/messages.po

```diff
@@ -210,11 +210,11 @@ msgstr ""
 "כמות התאונות הקטלניות, הקשות והקלות במקטע, לפי שנת התאונה, מחולק לפי "
 "חומרת התאונה, בפרק הזמן הנבחר."
 
-#: anyway/widgets/suburban_widgets/accident_count_by_car_type_widget.py:114
+#: anyway/widgets/suburban_widgets/accident_count_by_car_type_widget.py:134
 msgid ""
-"comparing vehicle type percentage in accidents in {} relative to national"
-" average"
-msgstr "השוואת אחוז הרכבים בתאונות במקטע {} לעומת ממוצע ארצי"
+"Comparing top 3 vehicle type percentage in accidents in {} relative to "
+"national average"
+msgstr "השוואת 3 המובילים באחוז הרכבים בתאונות במקטע {} לעומת ממוצע ארצי"
```

Comment:
Also - please verify title with product team - @MichalOren

Reply:
Hi @BarVolunteering and @atalyaalon 
Does the query include all the severity of the accident. Including light, severe & fatal?
Why "percent" while the data appears as a numbers?
In  general....it should look like this
![image](https://user-images.githubusercontent.com/61007501/154855437-27890331-d71c-4a91-ae59-d7f90def6aae.png)

As for the filter for **top 3** results it does not appear in the title but in the "info" section
So  insert the following sentence:
התפלגות תאונות לפי סוגי רכב במקטע, בפרק הזמן הנבחר. מוצגים 3 סוגי הרכב הנפוצים ביותר בתאונות במקטע זה.

Reply:
למעשה נראה שיש עוד הגדרות שחסרות. אבל אני לא רוצה לעכב את המשימה  שמוגדרת בתחילת הדף

Reply:
The query include all severities of the accident.
We are using percent because thats the original interface before the change, we can change it if you want.

What do mean in 
"As for the filter for top 3 results it does not appear in the title but in the "info" section"? - the information section in the JSON is not in use, do you mean we should use it?

I will change the title

Reply:
Hi Bar
- Keep current percentage values- no change.
- so the title probbly be
תאונות לפי סוג רכב  במקטע <> בהשוואה לכל הארץ
what do you think?

- kindly add the description above inside information section.  yes we should start using it in this widget
as i mentioned above this widget is missing some setting & definition

Reply:
1. בנוגע לכותרת "השוואת סוגי רכב המעורבים בתאונות במקטע <> בהשוואה לכל הארץ",  אין צורך לציין שאלו שלושת סוגי הרכבים עם הכי הרבה תאונות?
2. מה התוכן שאתם רוצים שיופיע בinformation ?
3. לא הבנתי את המשפט "this widget still missing some setting & definition", יש צורך להגדיר מחדש את המשימה?